klippy/extras/tmc2660.py

```diff
@@ -242,5 +245,20 @@ def cmd_SET_TMC_CURRENT(self, params):
             self.current = gcode.get_float('CURRENT', params, minval=CURRENT_MIN, maxval=CURRENT_MAX)
             self.set_current(self.current, self.printer.get_reactor().NOW)
 
+    cmd_DUMP_TMC_help = "Read and display TMC stepper driver registers"
+    def cmd_DUMP_TMC(self, params):
+        self.printer.lookup_object('toolhead').get_last_move_time()
+        gcode = self.printer.lookup_object('gcode')
+        for reg_name , val in zip(["DRVCONF", "DRVCTRL", "CHOPCONF", "SGCSCONF", "SMARTEN"],
+                            [self.reg_drvconf, self.reg_drvctrl, self.reg_chopconf, self.reg_sgcsconf, self.reg_smarten]):
+            msg = "%-15s %08x" % (reg_name + ":", val)
+            gcode.respond_info(msg)
```

Comment:
I think it would be preferable if the message made it clear that it was reporting a cached value.  Otherwise it might be confusing when compared to the behaviour of DUMP_TMC on the tmc2130/tmc2208.

-Kevin